static/js/index.js

```diff
@@ -87,6 +100,35 @@
       (event.keyCode < 91 || event.keyCode > 95)
     );
   };
+  
+    // Function which is used to traverse search links based on Up and down arrow key
+  var traverseElementOnKeyDown = function(event) {
+	
+	// Find active Element tabIndex
+	var tabIndexElement = document.activeElement.tabIndex;
+	
+	if(tabIndexElement < 8){
+		return;
+	}
+	// Based on Up and Down Key press move focus from next or previous search links
+	if(event.keyCode == 38)	{ // up key press
+		findElementByAttributeValue('tabIndex',--tabIndexElement).focus();
```

Comment:
I think we can use document.querySelector("[tabindex=N]") directly; We don't need JS support for IE<8